orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/cloudformation/DeployCloudFormationTask.java

```diff
@@ -89,8 +91,11 @@ public TaskResult execute(@Nonnull Stage stage) {
       }
     }
 
-    Map templateBody = (Map) task.get("templateBody");
-    if (templateBody == null || templateBody.isEmpty()) {
+    Object templateBody = task.get("templateBody");
+    if (templateBody instanceof Map) {
+      templateBody = new Yaml().dump(templateBody);
```

Comment:
It is necessary to add the `templateBody` back into the task map after translating to string?

Reply:
Yes :) Done.